src/main/kotlin/org/partiql/scribe/sql/SqlDialect.kt

```diff
@@ -89,6 +89,13 @@ abstract class SqlDialect : AstBaseVisitor<SqlBlock, SqlBlock>() {
             t = t concat ")"
             t
         }
+        node is Expr.BagOp -> {
```

Comment:
Copy over a change from the PLK PR https://github.com/partiql/partiql-lang-kotlin/pull/1506 since Scribe uses its own-defined `SqlDialect`.